### DISCUSSION TITLE
why I get this ?

### DISCUSSION THREAD

**Discussion by orestis119** [coder/code-server#2624] [General]
![image](https://user-images.githubusercontent.com/72334770/105547240-f5cd6e00-5d06-11eb-8627-90732a8d8b61.png)

**Comment by sr229** [comment]
I assume you're running this on a Windows host. Keep in mind code-server is only tested for Linux. Please use WSL2 instead if you plan to use Windows for your runtime.

**Reply by sr229** [reply]
For reference, see #2060.

**Comment by nooobcoder** [comment]
@orestis119 
You're on Windows CMD.EXE (from the error message). It **uses a different syntax to execute commands**. You'll need to use sh name.sh, assuming that you've got Cygwin or similar installed.

Windows uses the batch (**.bat**) file format to execute scripts and not **.sh** (shell). Please correlate accordingly as per your needs.


To clarify, Windows does not have a built-in utility to support .sh files. To run such, you'll need to install a third-party tool such as **Cygwin**.

Also do not keep this thread open, if your problem has been resolved, please mark the thread as answered. This would contribute to the organization a lot.

Thank You,
Have a great day.
